src/app/static/src/app/store/root/actions.ts

```diff
@@ -52,5 +57,27 @@ export const actions: ActionTree<RootState, RootState> & RootActions = {
 
     async changeLanguage(context, payload) {
         await changeLanguage<RootState>(context, payload)
+    },
+
+    async fetchADRKey(context) {
+        await api<RootMutation, RootMutation>(context)
+            .withSuccess(RootMutation.UpdateADRKey)
```

Comment:
Should include ignoreError to suppress console warning? Or do something on error?

Reply:
Probably should handle this error actually and show back to the user :+1: